R_program/MarkovWeatherFileGenerator.R

```diff
@@ -1,190 +1,30 @@
 #Markov script that generates two site-specific files (mkv_covar.in and mkv_prob.in) required by the markov weather generator in SOILWAT2
 
+#Three options for imputation of missing values due to insufficient weather data within function dbW_estimate_WGen_coefs via argument imputation_type: no imputation ("none"), imputation via mean using 5 values before and 5 values after the missing value ("mean"), and imputation by last-observation-carried-forward ("locf"). If the user wants to adjust the number of values used to calculate the mean, argument "imputation_span" must also be provided with an integer (ex: imputation_span = 3). See function dbW_estimate_WGen_coefs in rSOILWAT2 for additional details.
+
 #Set up system for parallel processing
 library(doParallel)
 registerDoParallel(proc_count)
 
   #Loop through all sites
   #load a particular site
-  site<-site[1] 
+  site<-site[1]
   s<-1
-  
-  #move to directory
-  setwd(paste("Site","_",site,sep=""))
+
   foreach (h = 1:H) %dopar%
     { #h = number of GCM X RCP X Times (scenarios)
     scen<-temp[h] #load a particular scenario
```

Comment:
I see that `scen` is used when printing the files, but is not passed to `dbW_estimate_WGen_coefs`. Do we not need to know the scenario to generate the coefficients?

Reply:
@chaukap the scenario (GCM x RCP x TimePeriod) is passed via the reference `sw_weatherList[[s]][[h]]` as h points to the particular scenario in sw_weatherList, despite not passing the name i.e. `scen` directly.

Reply:
@kpalmqui gotcha!